core/domain/summary_services.py

```diff
@@ -131,3 +131,18 @@ def _generate_query(categories):
         })
 
     return results
+
+
+def get_featured_explorations():
+    """Returns a list of featured explorations."""
+    exp_ids = []
+    exp_ids = exp_services.get_all_exploration_summaries()
```

Comment:
This function returns domain objects, not exploration ids. (A test would have caught this.)